msgraph-sdk-php.yml

```diff
@@ -0,0 +1,36 @@
+page_type: sample
+products:
+  - office-excel
+  - office-365
+  - office-outlook
+  - office-onedrive
+  - office-onenote
+  - office-planner
+  - office-sp
+  - ms-graph
+languages:
+  - php
+extensions:
+  contentType: sdks
+  technologies:
+    - Microsoft Graph
+    - Microsoft identity platform
```

Comment:
PHP library does not reference identity. Remove identity from the tech and services fields.

Reply:
Updated